_includes/tutorials/rekeying-function/ksql/markup/dev/make-build-file.adoc

```diff
@@ -0,0 +1,5 @@
+Create the following Gradle build file, named `build.gradle` for the project:
```

Comment:
A little more explanation could be used here about why we're making a Gradle file. Maybe good to note that KSQL supports Java for UDFs, and that we need some way to turn the files into an uberjar to give to KSQL.

Reply:
Done